app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -48,94 +48,108 @@ fun Home(
-                            .clip(backdropShape)
-                            .background(Backdrop)
-                    ) {
-                        Spacer(modifier = Modifier.size(24.dp))
-
-                        Text(
-                            text = "Trending Now",
-                            color = Text,
-                            fontSize = 14.sp,
-                            fontFamily = manropeFamily,
-                            fontWeight = FontWeight.Bold,
-                            modifier = Modifier.padding(start = 24.dp)
+            AnimiteTheme {
+                Box(
+                    modifier = Modifier
+                        .verticalScroll(rememberScrollState())
+                        .navigationBarsPadding()
+                ) {
+                    Box {
+                        Image(
+                            painter = painterResource(id = Res.drawable.background),
+                            contentDescription = "Background",
+                            modifier = Modifier.fillMaxWidth()
                         )
 
-                        Spacer(modifier = Modifier.size(12.dp))
-
-                        MediaSmallRow(
-                            mediaList = trendingNowMediaList,
-                            onItemClick = { itemId ->
-                                navigator.navigate(
-                                    MediaPageDestination(
-                                        id = itemId,
-                                        mediaTypeArg = homeMediaType.rawValue
+                        Box(
+                            modifier = Modifier
+                                .background(
+                                    Brush.verticalGradient(
+                                        listOf(
+                                            Color.Transparent,
+                                            MaterialTheme.colorScheme.secondaryContainer.copy(
+                                                alpha = 0.5f
+                                            )
+                                        )
                                     )
-                                ) {
-                                    launchSingleTop = true
-                                }
-                            }
-                        )
-
-                        Spacer(modifier = Modifier.size(24.dp))
+                                )
+                                .fillMaxWidth()
+                                .aspectRatio(1f)
+                        ) {  }
 
                         Text(
-                            text = "Popular This Season",
-                            color = Text,
-                            fontSize = 14.sp,
-                            fontFamily = manropeFamily,
-                            fontWeight = FontWeight.Bold,
-                            modifier = Modifier.padding(start = 24.dp)
+                            text = "おかえり",
```

Comment:
Hardcoded text 😟
Do I need to open an issue for string-ifying things?

Reply:
No :fearful: